<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>425</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MYC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QPushButton" name="viewCredits">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>View Credits</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Firmware</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="activeFirmware">
         <property name="text">
          <string>Active Firmware:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="activeFirmwareValue">
         <property name="text">
          <string>Unknown</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="latestFirmware">
         <property name="text">
          <string>Latest Firmware: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="latestFirmwareValue">
         <property name="text">
          <string>Unknown</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="firmwareVersions">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="updateFirmware">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Update Firmware</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="autoUpdateFirmware">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Auto-update Firmware</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Keys</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="titleKeys">
         <property name="text">
          <string>title.keys: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="titleKeysValue">
         <property name="text">
          <string>Not Found</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="prodKeys">
         <property name="text">
          <string>prod.keys: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="prodKeysValue">
         <property name="text">
          <string>Not Found</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="updateKeys">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Update Keys</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="autoUpdateKeys">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Auto-update Keys</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Yuzu</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" colspan="3">
        <widget class="QPushButton" name="yuzuInstaller">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Yuzu Installer</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="yuzuEA">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Start Yuzu EA</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="yuzu">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Start Yuzu</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QComboBox" name="yuzuSelector">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QCheckBox" name="autoStart">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Auto-start after updates</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
